src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/GitLabContainer/create_user.rb

```diff
@@ -0,0 +1,13 @@
+user = User.create();
+user.name = 'test';
```

Comment:
Hardcoded, what if this is called by tests in parallel? Is gonna fail. Maybe you can try what is described here https://www.codecademy.com/article/ruby-command-line-argv

Reply:
> what if this is called by tests in parallel

then each test should have its own instance of the container so it doesn't matter (unless we are running multiple tests in the same JVM at once which we are not (threading as opposed to forking))